owslib/wps.py

```diff
@@ -505,21 +489,17 @@ def __init__(self, version=WPS_DEFAULT_VERSION, verbose=False, timeout=None, aut
         super(WPSCapabilitiesReader, self).__init__(
             version=version, verbose=verbose, timeout=timeout, auth=auth)
 
-    def readFromUrl(self, url, username=None, password=None,
-                    headers=None, verify=True, cert=None):
+    def readFromUrl(self, url, headers=None):
```

Comment:
Should really issue a `DeprecationWarning` here for some time before removing existing functionality to avoid breaking code

```python
def readFromUrl(self, url, username=None, password=None, headers=None, verify=True, cert=None):
    import warnings
    message = 'The use of "username", "password", "verify", and "cert" is deprecated. ' + \
        'Please use the "auth" keyword during class instantiation. ' + \
        'These keywords will be removed in a future release.'
    warnings.warn(message, DeprecationWarning)
    # rest of the method
```

Reply:
I can add the deprecation warning in this PR. 

I would have considered the `WPSReader` class as part of the internal interface. Though this is not obvious from the code.